From 318b0fd7afe42b35f150eceebe54b95309037b39 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Wed, 8 Jul 2015 12:37:47 +0100
Subject: [PATCH] automake: Do not install developer files on end-user systems.

autogen.sh, .dir-locals.el, .vimrc, .ycm_extra_conf.py, .travis.yml,
.mailmap files are only useful with the source tree, for the
developers. Do not install these files as documentation on the
end-user systems, but keep them distributed with the tarball.
---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 19a3706..603a479 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -647,7 +647,9 @@ dist_doc_DATA = \
 	DISTRO_PORTING \
 	src/libsystemd/sd-bus/PORTING-DBUS1 \
 	src/libsystemd/sd-bus/DIFFERENCES \
-	src/libsystemd/sd-bus/GVARIANT-SERIALIZATION \
+	src/libsystemd/sd-bus/GVARIANT-SERIALIZATION
+
+EXTRA_DIST += \
 	autogen.sh \
 	.dir-locals.el \
 	.vimrc \
-- 
2.4.5

